index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alvaro's Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <nav class="navbar">
+            <div class="logo_nav">
+                <a href="#"><img class="image_nav" src="images/my_logo.svg" alt=""></a>
+            </div>
+            <a href="#" ><img class="menu_icon" src="images/menu.svg" alt=""></a>
+        </nav>
+        <section class="first_section">
+            <h1 class="title content">I’m Alvaro. Glad to see you!</h1>
+            <p class="paragraph content">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.</p>
+            <div class="connect content">
+                <a href="#" class="connect_anchor button">
+                    <h3>LET’S CONNECT</h3>
+                </a>
+            </div>
+                
+            <ul class="social_icons content">
+                <li><a href="#" class="social_icon"><img class="icon" src="images/twitter_icon.svg" alt=""></a></li>
+                <li><a href="#" class="social_icon"><img class="icon" src="images/linkedin_icon.svg" alt=""></a></li>
+                <li><a href="#" class="social_icon"><img class="icon" src="images/medium_icon.svg" alt=""></a></li>
+                <li><a href="#" class="social_icon"><img class="icon" src="images/github_icon.svg" alt=""></a></li>
+                <li><a href="#" class="social_icon"><img class="icon" src="images/hand_icon.svg" alt=""></a></li>
```

Comment:
- [optional] Please kindly add the `alt` attribute for the `img` elements, the `alt` attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the `src` attribute, or if the user uses a screen reader). Please take note of line 19 and 21 as well. Thanks. ℹ️